IIPS/iip-015.md

```diff
@@ -0,0 +1,48 @@
+---
+iip: 15
+title: IndexCoop Shield Mining
+status: Proposed
+author: Luciano (tw @Luciano_vPEPO)
+discussions: https://gov.indexcoop.com/t/iip-015-indexcoop-shield-mining/660/15
+created: 2021-01-04
+---
+
+## Simple Summary
+
+Enable INDEX Shield Mining incentives for the IndexCoop protocol for 10 weeks with an issuance of 0.05 INDEX per NXM per week targeted (target 12,000 INDEX total or 1,000 INDEX per week to attract 40,000 NXM enabling about 1.6M USD worth of covers to be bought at a reasonable rate of ~6% per year)
+
+## Abstract
+
+Allocate 12,000 INDEX towards Nexus Mutual’s Shield Mining program for the IndexCoop smart contracts with the following parameters:
```

Comment:
Which smart contracts do we exactly want covered?
You mentioned DPI smart contract and potentially staking rewards?
That should be included in here.